docs/content/design/broker.md

```diff
@@ -63,3 +63,12 @@ Returns the metrics of the datasource.
 
 Returns a flag indicating if the broker knows about all segments in Zookeeper. This can be used to know when a broker node is ready to be queried after a restart.
 
+* `/druid/v2/candidates/?datasource=<datasource-name>&intervals=<comma-separated-intervals-in-ISO8601-format`
```

Comment:
addressed comment. now it's `/druid/v2/datasources/{dataSourceName}/intervals/{comma-separated-intervals-in-ISO8601-format}`